docs/configuration/configuration.md

```diff
@@ -2705,6 +2705,94 @@ tls_config:
+* `__meta_oci_private_ip` : The private ip address of primary vnic for the instance.
+* `__meta_oci_public_ip` : The public ip address of primary vnic for the instance, if present.
+* `__meta_oci_compute_ocpus` : The OCPU count for the instance.
+* `__meta_oci_compute_processor_description` :The processor description for the instance.
+* `__meta_oci_compute_memory_in_gbs` : The amount of RAM in GBs.
+* `__meta_oci_compute_instance_shape` : The OCI compute [shape](https://docs.oracle.com/en-us/iaas/Content/Compute/References/computeshapes.htm) for the instance.
+* `__meta_oci_compute_defined_tag_<tagkey>` : Each [defined tag](https://docs.oracle.com/en-us/iaas/Content/Tagging/Concepts/taggingoverview.htm#concepts) value of the instance.
+* `__meta_oci_compute_freeform_tag_<tagkey>` : Each [freeform tag](https://docs.oracle.com/en-us/iaas/Content/Tagging/Concepts/understandingfreeformtags.htm) value of the instance.
+
+See below for the configuration options for *OCI Compute* discovery:
+
+```yaml
+# The OCID, the unique ID about your OCI tenancy/account. Always required.
+tenancy_ocid: <string> 
+# The OCI region code, 
+# as per https://docs.oracle.com/en-us/iaas/Content/General/Concepts/regions.htm. This determines the OCI API endpoint used by OCI Compute SD. Always required.
+region: <string> 
+# The OCID for the compartment for your compute instances to be scraped.
+compartment_ocid: <string>
+# The port to scrape metrics from. 
+[ port: <int> | default = 80 ]
+# Refresh interval to re-read the instance list.
+[ refresh_interval: <duration> | default = 300s ]
+
+# Optional `authencation` mechanism to be used by OCI Compute SD.
+authencation:
+  # Sets the authentication type. Permissible enum values: InstancePrincipal or UserPrincipal
+  [ type: <enum> | default: InstancePrincipal ]
+
+  # In case UserPrincipal is used, path to your local OCI Config file.
+  [ oci_config_file_path: <string> | default: ~/.oci/config]
+  # In case UserPrincipal is used, User profile from OCI Config file.
+  [ oci_config_file_profile: <string> | DEFAULT ]
+  # In case UserPrincipal is used, password for your OCI Config file.
+  [ oci_config_file_pvt_key_password: <secret> | default: "" ]
+
+# Tag Filters can be used optionally to filter the instance list, as per the tags applied to them 
+# Only instances with 'all' the tags will pass the filter criteria. If instance has more than the following tags, it will pass the filter criteria.
+tag_filters:
+  [ - tagNamespace: <string> # only needed for defined tags, else can be skipped
+      tagKey: <string>
+      tagValue: <string>, [...] ]
+```
+
+#### OCI IAM policies for authorization 
+When using *InstancePrincipal*, for authentication with OCI APIs, the Prometheus needs to be running on OCI Compute Instance. Additionally compute instance must belong to a dynamic group say *dg_prometheus* with following OCI IAM policies, assigned to the dynamic group. 
+```
```

Comment:
Add an blank line before this

Reply:
got it before
"#### OCI IAM policies for authorization"